internal/cltest/cltest.go

```diff
@@ -82,6 +82,7 @@ func NewConfigWithWSServer(wsserver *httptest.Server) *TestConfig {
 			EthGasBumpThreshold:      3,
 			EthGasPriceDefault:       *big.NewInt(20000000000),
 			DatabasePollInterval:     store.Duration{Duration: time.Millisecond * 500},
+			AllowOrigins:             "http://localhost:3000,http://localhost:6689",
```

Comment:
When do we need this for tests?

Reply:
https://github.com/smartcontractkit/chainlink/pull/399/files#diff-2b0d4449457bb40cb9566b905c1b12f6

The `envDefault`'s don't get picked up in tests. Unless I'm doing something wrong...

Reply:
That sounds right. I wasn't sure if we even needed them, but it looks like the set values are used in `TestCors_OverrideOrigins`.